Makefile

```diff
@@ -75,3 +75,23 @@ testlxd:
 testmicrok8s:
 ## testmicrok8s: Run unit tests against microk8s
 	TF_ACC=1 TEST_CLOUD=microk8s go test ./... -v $(TESTARGS) -timeout 120m
+
+.PHONY: install-snap-dependencies
+install-snap-dependencies:
+## install-snap-dependencies: Install snap dependencies for terraform-provider-juju
+	@echo "Installing snap dependencies"
+ifneq ($(HAS_TERRAFORM),)
+	@echo "Refreshing terraform snap"
+	@sudo snap refresh terraform --classic
```

Comment:
Why are we doing this? The snap should refresh automatically as needed.

Reply:
I thought that without it, we would just receive the following output:

```
snap "terraform" is already installed, see 'snap help refresh'
```

Reply:
ok, I'll rewrite this logic... I saw something like this in our previous Makefiles

Reply:
I'll rewrite this login in more flexible way